packages/labextension/style/index.css

```diff
@@ -1,132 +1,145 @@
-}
-
-.nbdime-Diff .nbdime-header-remote {
-  display: inline-block;
-  width: calc(49% - 8px);
-  background-color: var(--jp-diff-added-color0);
-  padding: 0px 4px;
-  vertical-align: top;
-}
-
-.nbdime-Diff .nbdime-header-local {
-  display: none;
-  width: calc(49% - 8px);
-  padding: 0px 4px;
-  vertical-align: top;
-}
-
-/* Hiding unchanged cells if told to */
-.nbdime-root.jp-mod-hideUnchanged .jp-Cell-diff.jp-Diff-unchanged {
-  display: none;
-}
-
-/* Show a marker with the number of cells hidden before */
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Cell-diff[data-nbdime-NCellsHiddenBefore]::before,
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Diff-addremchunk[data-nbdime-NCellsHiddenBefore]::before {
-  content: attr(data-nbdime-NCellsHiddenBefore) ' unchanged cell(s) hidden';
-  position: absolute;
-  width: 100%;
-  top: 0;
-  background-color: var(--jp-layout-color2);
-  border-top: solid var(--jp-layout-color3) 1px;
-  border-bottom: solid var(--jp-layout-color3) 1px;
-  text-align: center;
-}
-
-/* Show a marker with the number of cells hidden after (for hidden cells at end) */
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Cell-diff[data-nbdime-NCellsHiddenAfter]::after,
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Diff-addremchunk[data-nbdime-NCellsHiddenAfter]::after {
-  content: attr(data-nbdime-NCellsHiddenAfter) ' unchanged cell(s) hidden';
-  position: absolute;
-  width: 100%;
-  bottom: 0;
-  background-color: var(--jp-layout-color2);
-  border-top: solid var(--jp-layout-color3) 1px;
-  border-bottom: solid var(--jp-layout-color3) 1px;
-  text-align: center;
-}
-
-.nbdime-root.jp-mod-hideUnchanged .jp-Cell-diff[data-nbdime-NCellsHiddenBefore],
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Diff-addremchunk[data-nbdime-NCellsHiddenBefore] {
-  padding-top: 40px;
-}
-
-.nbdime-root.jp-mod-hideUnchanged .jp-Cell-diff[data-nbdime-NCellsHiddenAfter],
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Diff-addremchunk[data-nbdime-NCellsHiddenAfter] {
-  padding-bottom: 40px;
-}
-
-/* Marker for when all cells are unchanged and hidden */
-.nbdime-root.jp-mod-hideUnchanged
-  .jp-Notebook-diff[data-nbdime-AllCellsHidden]::after {
-  content: 'No changes, ' attr(data-nbdime-AllCellsHidden)
-    ' unchanged cell(s) hidden';
-  display: block;
-  width: 100%;
-  background-color: var(--jp-layout-color2);
-  border-top: solid var(--jp-layout-color3) 1px;
-  border-bottom: solid var(--jp-layout-color3) 1px;
-  text-align: center;
-}
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+ @import url('nbdime/lib/common/collapsible.css');
+ @import url('nbdime/lib/upstreaming/flexpanel.css');
+ @import url('nbdime/lib/common/dragpanel.css');
+ @import url('nbdime/lib/styles/variables.css');
+ @import url('nbdime/lib/styles/common.css');
+ @import url('nbdime/lib/styles/diff.css');
+ @import url('nbdime/lib/styles/merge.css');
+
+ .nbdime-Widget {
+   display: flex;
+   flex-direction: column;
+ }
```

Comment:
For me, this file renders as a full diff, even though there is very little changes. Is there a change to line endings or something? 😕

Reply:
Actually, I think there is an addition of 1 space indent in this file now?

Reply:
@vidartf Both the comma and the indentation issues have been fixed in commit

Reply:
> Actually, I think there is an addition of 1 space indent in this file now?

Yes there was. Introduced with a copy-paste and I did not see indentation was changed.